docs/using-mycroft-ai/customizations/languages/german.md

```diff
@@ -0,0 +1,286 @@
+---
+description: >-
+  Dies ist eine Anleitung wie Mycroft in Deutsch eingerichtet werden kann. Diese Seite beinhaltet auch eine Kurzanleitung.
+---
+   <sub><sub>TTS = Sprachoutput  / STT = Sprachinput    / WakeWord = Initiierung </sub></sub>
+---
+
+# Mycroft auf deutsch
+
+## Erster Einstieg
+
+!!!Aktuell ist das paaren von Mycroft noch nicht vollständig in deutsch möglich, daher sollten sie Mycroft bereits unter
+"lang": "en-us" verbunden und eingerichtet haben.
+
+Zuerst, [verbinden Sie sich mit ihrem Gerät (z.B. SSH)](https://mycroft.ai/documentation/mark-1/#connecting-to-the-mark-1-via-ssh).
+
+Folgend wird an den Beispielen Google/Mbrola der Sprachoutput (TTS) auf deutsch gesetzt. Mycroft bietet weitere TTS-Möglichkeiten über [marytts](http://mary.dfki.de/), [responsive_voice](https://responsivevoice.org/), [polly](https://aws.amazon.com/de/polly/), auf die hier jedoch nicht näher eingegangen wird.
+
+## Text-to-Speech (TTS) über Google
+
+
+Erstelle oder ändere die eigene `~/.mycroft/mycroft.conf` Konfigurationsdatei wie folgt.
```

Comment:
I try now to encourage people to use the [Configuration Manager tool](https://mycroft-ai.gitbook.io/docs/using-mycroft-ai/customizations/config-manager), rather than editing these files directly.

Simple changes can be done in one command: `mycroft-config set x y`
Or you can edit the file: `mycroft-config edit user`

But editing using this tool will validate the changes when you save them. So if you have missed a comma or something, it will be flagged as an issue rather than just not working.